toolchain/check/impl.cpp

```diff
@@ -218,6 +222,7 @@ auto ImplWitnessForDeclaration(Context& context, const SemIR::Impl& impl)
         break;
     }
   }
```

Comment:
This loop doesn't really seem to be doing anything any more. It does `LoadIImportRef`, but we could do that later, and otherwise is just deciding whether each table entry is `Invalid` or `Error`. I wonder if we can just fill the table with `InstId::Invalid` here, and fill in any slots that should be `Error` in `FinishImplWitness` instead?

Reply:
(It's fine to defer this to a later PR or just add a TODO.)

Reply:
I changed this loop to just do `LoadImportRef` since there wasn't a clearly better place, and moved the rest to `FinishImplWitness`